collection-scripts/gather_monitoring

```diff
@@ -1,29 +1,208 @@
 #!/bin/bash
```

Comment:
@mtulio based on [comment from Damien](https://github.com/openshift/must-gather/pull/234#issuecomment-865167716), do you think renaming this script so that its usage would be to gather additional data that can be gathered on demand? 
 
```
oc adm must-gather -- /usr/bin/gather-monitoring-mertics
```
I also wonder if additional parameters can be passed to the script. e.g. 

```
oc adm must-gather -- /usr/bin/gather-monitoring-mertics --start-date='..' --end-date='...' --step='...'
```


cc: @dgrisonnet

Reply:
> based on comment from Damien, do you think renaming this script so that its usage would be to gather additional data that can be gathered on demand?

@sthaha yes, it could be a good idea rename it to keep the roles separated from default "monitoring" collector. 

> I also wonder if additional parameters can be passed to the script. e.g.

If we will rename the script to run on demand, it will be a great idea, so we can avoid some extra arguments using CM - not sure if we can remove the CM due to long parameters that can be sent with long list of metrics names.